### ISSUE TITLE
Crete PSP "Users and Groups"

### ISSUE THREAD

**Issue by flavio** [kubewarden/policy-hub#28]
Create a Kubewarden policy that implements the [Users and Groups PSP](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#users-and-groups).

This is a mutating policy.

### Configuration

**TODO:** we should mimic what the PSP offers.

**Comment by jvanz** [comment]
I just realized that the SDK that I used to start to code the policy ( golang SDK) does not support mutating requests. :face_palm: 

I need to use a different programming language or check if it is possible to update the SDK to add this feature.

**Comment by flavio** [comment]
> I need to use a different programming language or check if it is possible to update the SDK to add this feature.

Go policies cannot do mutations due to limitations with TinyGo. We have to wait for proper handling of marshalling and unmarshalling

**Comment by jvanz** [comment]
> Go policies cannot do mutations due to limitations with TinyGo. We have to wait for proper handling of marshalling and unmarshalling

Yeah... I remember you talking about the limitations when I try to do the mutations. Sorry :disappointed:

**Comment by jvanz** [comment]
https://github.com/kubewarden/psp-user-group
